spec/site_prism/page_objects/pages/application.rb

```diff
@@ -35,7 +35,9 @@ def app_sections
         cases_new_sar:                  'Cases::New::SARPage',
         cases_new_offender_sar_subject_details:                 'Cases::New::OffenderSARPageSubjectDetails',
         cases_new_offender_sar_requester_details:                 'Cases::New::OffenderSARPageRequesterDetails',
+        cases_new_offender_sar_recipient_details:                 'Cases::New::OffenderSARPageRecipientDetails',
         cases_new_offender_sar_requested_info:                 'Cases::New::OffenderSARPageRequestedInfo',
+        cases_new_offender_sar_request_details:                 'Cases::New::OffenderSARPageRequestDetails',
```

Comment:
Could this be more alphabetical? Small thing no reason to change if you don't think its necessary 😄

Reply:
Good point - we try to keep things alphabetical everywhere else, will change 👍